app/src/main/java/com/edu/movie/screen/moviedetails/adapter/CastsAdapter.kt

```diff
@@ -0,0 +1,64 @@
+package com.edu.movie.screen.moviedetails.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.edu.movie.R
+import com.edu.movie.data.model.Cast
+import com.edu.movie.utils.Constant
+import com.edu.movie.utils.LoadImageFromUrl
+import kotlinx.android.synthetic.main.item_cast.view.*
+
+class CastsAdapter : RecyclerView.Adapter<CastViewHolder>() {
+
+    private var onItemCastClickListener: ((Int) -> Unit)? = null
```

Comment:
sao không gán giá trị mặc định not null luôn  để đỡ phải check null 
```suggestion
    private var onItemCastClickListener: (Int) -> Unit = {}
```